src/components/Summary/AlterationSummary.vue

```diff
@@ -266,7 +262,7 @@ export default class AlterationSummary extends Mixins(
   }
 
   get originalEntityType (): string {
-    return this.getOriginalSnapshot?.businessInfo?.legalType
+    return this.getBusinessSnapshot?.businessInfo?.legalType
```

Comment:
This name change makes sense.. since there is no 'Original' per se. It's all current! 👍